docs/Config_Changes.md

```diff
@@ -8,6 +8,20 @@ All dates in this document are approximate.
 
 ## Changes
 
+20230801: The setting `fan.off_bellow` has been changed to `fan.min_power`.
+However, this change will not affect users who do not utilize this setting.
+With this update, a PWM scaling has been introduced between `min_power` and
+`max_power`. Fans that require a higher `min_power` will now have access to
+their full "safe" power curve. By properly setting `min_power`, any fan
+(such as CPAP) should engage even at `M106 S1`. It's recommended to review
+your slicer/macros to adjust your fan speeds. Your previously designated 20%
+fan speed may no longer represent your lowest fan setting but will now
+correspond to an actual 20% fan speed.
+If you had previously set `max_power` to anything below 1.0 (default),
+it is advisable to recalibrate `min_power` again, with the settings
+`min_power=0` and `max_power=1`.
```

Comment:
Should probably be `min_power: 0` and `max_power: 1` to keep in line with klipper formatting and not confuse users.

Reply:
Ah thanks. Will update